torch/_inductor/mkldnn.py

```diff
@@ -351,7 +353,9 @@ def __init__(self, linear: nn.Module, input_size: list):
 
     def _update_module_params(self, linear, input_size):
         self.__dict__ = copy.deepcopy(linear.__dict__)
-        self.batch_size = int(reduce(lambda x, y: x * y, input_size) / input_size[-1])
+        self.batch_size = sym_int(
```

Comment:
`FloorDiv(reduce(...))`? Or `reduce(..., input_size[:-1])`? Converting truediv to int wasn't right before.

Reply:
Making it `reduce(..., input_size[:-1])` sounds simpler.